### PR TITLE
feat(mini.indentscope): add scope color

### PR DESCRIPTION
**Pull request by Pandoks**
Added feature to let users choose what color they want for their scope indent line.

#585 but for mini's indentscope

### CODE REVIEW

File: lua/catppuccin/groups/integrations/mini.lua [hunk 1]
```diff
@@ -6,13 +6,20 @@ function M.get()
...
+	end
+
 	return {
 		MiniCompletionActiveParameter = { style = { "underline" } },
 
 		MiniCursorword = { style = { "underline" } },
 		MiniCursorwordCurrent = { style = { "underline" } },
 
-		MiniIndentscopeSymbol = { fg = C.text },
+		MiniIndentscopeSymbol = indentscope_color,
```
Comment (mrtnvgr) [inline lua/catppuccin/groups/integrations/mini.lua:22]:
```suggestion
		MiniIndentscopeSymbol = { fg = C[indentscope_color] and C[indentscope_color] or C.text },
```

File: lua/catppuccin/groups/integrations/mini.lua [hunk 2]
```diff
@@ -6,13 +6,20 @@ function M.get()
 
 	local inactive_bg = transparent_background and "NONE" or C.mantle
 
+	local indentscope_color = O.integrations.mini.indentscope_color
+	if C[indentscope_color] then
+		indentscope_color = { fg = C[indentscope_color] }
+	else
+		indentscope_color = { fg = C.text }
+	end
+
```
Comment (mrtnvgr) [inline lua/catppuccin/groups/integrations/mini.lua:15]:
```suggestion
	local indentscope_color = O.integrations.mini.indentscope_color
```

**Review by mrtnvgr — APPROVED** [review]
Thanks for your contribution 🙂

**Comment by nullchilly** [comment]
> **Note**
> Can be simplified further to:

```lua
fg = C[indentscope_color] or C.text
```

**Pull request merged**